packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -262,6 +306,11 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
       legendText: '',
       value: enrollmentLocation,
     },
+    {
+      style: { width: '50%' },
+      legendText: '',
+      value: stateSelect,
+    },
```

Comment:
```suggestion
    config.programStatus && {
      style: { width: '50%' },
      legendText: '',
      value: stateSelect,
    },
```

or something to that effect...